backend/src/main/resources/application.yaml

```diff
@@ -17,6 +17,11 @@ config:
   # CORS header, should be the User Management Console:
   allowed-origins: "http://localhost:8080"
 spring:
+  datasource:
+    url: jdbc:oracle:thin:@daudelin.hlth.gov.bc.ca:1521:kcd
+    username: keycloak 
+    password: ${oracle.dev-password}
```

Comment:
Please submit a request to the DBA team for a proxy account for DB access. It will be read-only and have permissions only on those tables it needs to access to.